platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java

```diff
@@ -405,6 +407,29 @@ public void tagDelete(TagDeleteRequest request) throws ApiException {
         return result;
     }
 
+
+    @Path("config")
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Map<String, String> setConfig(ConfigRequest request) throws ApiException {
```

Comment:
It is not necessary to return a map because it returns the same that we requested. It would be better to make a `void` method